src/main/java/edition/academy/seventh/serivces/BooksService.java

```diff
@@ -0,0 +1,30 @@
+package edition.academy.seventh.serivces;
+
+
+import edition.academy.seventh.repositories.BooksRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edition.academy.seventh.database.models.Book;
+import java.util.List;
+
+/**
+ * Service which connect with repository {@link BooksRepository} to persist books in PostgreSQL
```

Comment:
Dokumentacja zupełnie nie pasuje do odpowiedzialności klasy. To jest po prostu serwis udostępniający API do zapisu książek do bazy. Nie ma znaczenia z czym się łączy i co ta klasa będzie robić. Implementacje i zależności widać przecież w ciele klasy